packages/remix-dev/__tests__/create-test.ts

```diff
@@ -266,7 +266,7 @@ describe("the create command", () => {
       "create",
       projectDir,
       "--template",
-      "https://example.com/remix-stack.tar.gz",
+      "https://github.com/private-org/private-repo/releases/download/v0.0.1/stack.tar.gz",
```

Comment:
Are you sure about this change?

Reply:
Yeah, I think so. This test is for `succeeds for private github release tarballs when including token`, which should be a github url instead of `example.com`:
https://github.com/remix-run/remix/blob/6ccac77e14d240cd8eb5b213021f08eb9cbb32a0/packages/remix-dev/__tests__/create-test.ts#L263-L284

We test `example.com` (correctly) here:
https://github.com/remix-run/remix/blob/6ccac77e14d240cd8eb5b213021f08eb9cbb32a0/packages/remix-dev/__tests__/create-test.ts#L228-L245